Analysis/Smells/LineSmells/LineTooLong.cs

```diff
@@ -0,0 +1,14 @@
+namespace BSDetector {
+    public class LineTooLong : LineSmell {
+        public LineTooLong() {
+            SmellName = "Line too long";
+            SmellDescription = "Lines that are too long make your code less readable.";
+        }
+        public override void AnalyzeLine(string currentLine, string previousLine, int lineNum) {
+            if (currentLine.Length > 140)
+            {
+                RegisterOccurance(lineNum, 0, lineNum, currentLine.Length - 1);
```

Comment:
ColStart smallest value is 1, not 0. Consequently, you shouldn't subtract 1 from line length.